etc/sloc_translate.py

```diff
@@ -66,12 +75,14 @@ def sloc_translate(args):
         (filename, total_sloc, physical_loc) = file_sloc(path, verbose=True)
         print(f"{filename}, {total_sloc}, {physical_loc}")
     elif len(args) == 3:
-        cleaned = [f".{x}" for x in args[2].split(',')]
+        cleaned = [f".{x}" for x in args[2].split(",")]
         walk_sloc(args[1], cleaned, verbose=True)
     else:
-        print("Expected either 1 argument (a single file to parse" +
-              " and print) or 2 (a directory root & file pattern)")
+        print(
```

Comment:
Should this really be a `print`, `warning`, or exception?

Reply:
I think it should really be an error, but I'm reluctant to spend time fixing this script.  The `etc/` directory is really a dumping ground for experimental functionality, and I'd like to remove it entirely in a future release.

This script doesn't currently import any of the logging stuff (and isn't called in a way that would configure the logging correctly) so fixing this properly would affect more than just this line.  If we decide that a SLOC-counting tool is important, I'd rather reintroduce the functionality as a new script in `bin/` that is actually documented, tested, and part of the package.